lib/install/validate/validator.go

```diff
@@ -658,3 +663,48 @@ func (v *Validator) AddDeprecatedFields(ctx context.Context, conf *config.Virtua
 
 	return &dconfig
 }
+
+func (v *Validator) syslog(conf *config.VirtualContainerHostConfigSpec, input *data.Data) {
```

Comment:
at least, we could move the url parsing to cmd package, and leave the schema, port detection here.